### DISCUSSION TITLE
modifying admonitions

### DISCUSSION THREAD

**Discussion by chipbrock** [executablebooks/meta#390] [Q&A]
I would like to change some of the default admonition icons, colors, and especially titles.

As an example, I'd like to change the Danger admonition to say "Example" instead of "Danger" and change the icon to the same one used for "important" (the circled "!"). 

 I assume that this should be doable in css and I see some of it in the theme.css, but I don't seem to be able to change them if I put different, say, icon in a separate css file inside of _static. I can change other aspects of rendering in that same private css file.

So instead of `  --pst-icon-admonition-danger: var(--pst-icon-exclamation-triangle);`

I make it `  --pst-icon-admonition-danger: var(--pst-icon-exclamation-circle);`

It still renders as the stock danger admonition. 

Is this doable? More importantly, is changing the title doable? Danger, Error, Caution are not useful to me and I would like to repurpose their appearances. 

I realize I can use the general admonition and specify the title, but they would all be blue and I'd like different of my new admonitions to be different colors. 

Thanks.

**Comment by choldgraf** [comment]
you could try using the general admonition along with a "class" argument as a non-hacky solution.

e.g.:

````
```{admonition} My title
:class: warning

This admonition will be styled as a warning
```
````

**Reply by chipbrock** [reply]
Thanks.
That does just about what I wanted. Is there a place to see all of the class definitions? I might have puzzled that out if I knew how to find the definitions. Theme.css doesn't seem to do it and there are a few other css files that are pretty indecipherable as they lack line-structure. 

You know what would be really neat? (thinking with my current need), would be a numbered admonition that would handle counters like LaTeX. So each successive "Example" admonition would get a number and a reference label. 

Anyhow, thanks, again. This project seems so much more project-like than Bookdown, which really looked like the exhausted effort of one heroic guy. I always feared that I was one update away from work-stopping and no actual community.

**Reply by chipbrock** [reply]
Your suggest is nearly there! Thanks. 

Is there a way to change the font-family inside of an admonition class?

I tried:

.admonition.danger { 
 	font-size: 80%;
     font-family: "Lucida Sans Unicode", "Lucida Grande", sans-serif;
    color: #b94a48;
    background-color: #e0e8f3;
    border-color: #eed3d7;
}

Where the colors are arbitrary just to see if I affected anything. I did, but the attempt at the font family failed (I use a serif font overall:

html {
font-family: "Palatino Linotype", "Book Antiqua", Palatino, serif;
}

)

It would be great to see the overall css files for MyST and/or Sphinx so I could figure out what to change.  Not knowing which drives this bus.
